com.unity.renderstreaming/Documentation~/en/tutorial.md

```diff
@@ -1,17 +1,9 @@
 # Tutorial
 
-Currently, there are two samples:
+In this tutorial, we introduce the way to use samples published on Package Manager.
+If you want to have a tutorial for the template, please see the document on the [GitHub repository](https://github.com/Unity-Technologies/UnityRenderStreaming).
```

Comment:
~If you want to have a tutorial for the template,~

For template installation tutorial,